src/coord/src/catalog/builtin.rs

```diff
@@ -949,6 +949,19 @@ lazy_static! {
         // for this to be persisted.
         persistent: true,
     };
+    pub static ref MZ_ACCESS_METHODS: BuiltinTable = BuiltinTable {
+        name: "mz_ams",
+        schema: MZ_CATALOG_SCHEMA,
+        desc: RelationDesc::empty()
+                .with_named_column("oid", ScalarType::Oid.nullable(false))
+                .with_named_column("amname", ScalarType::String.nullable(false))
+                .with_named_column("amhandler", ScalarType::RegProc.nullable(false))
+                .with_named_column("amtype", ScalarType::Char{length: Some(1)}.nullable(false))
+                .with_key(vec![0]),
+        id: GlobalId::System(4049),
+        index_id: GlobalId::System(4050),
+        persistent: false,
+    };
```

Comment:
The `mz_catalog` tables are meant to be Materialize's "native" source of truth about what's in the catalog. Since we don't keep track of any access methods, we shouldn't introduce this table since it'll require documentation etc. The goal is just to support `pg_am`, right? If so, doing it all with a view should do the trick!

```sql
CREATE VIEW pg_am AS
SELECT
    NULL::oid AS oid,
    NULL::text AS amname,
    NULL::regproc AS amhandler,
    NULL::char(1) AS amtype
WHERE false
```

Reply:
good call! Will do.